test/core/binary.wast

```diff
@@ -176,7 +862,7 @@
     "\1a"                      ;; drop
     "\0b"                      ;; end
   )
-  "zero flag expected"
+  "zero byte expected"
```

Comment:
Fizzy spectest runner uses these strings only for logging, so this is not much useful for us, but also doesn't hurt, so I included such changes to be closer to master and make future backporting potentially easier.